lib/adyen/services/checkout.rb

```diff
@@ -53,4 +85,48 @@ def result(request, headers = {})
       @client.call_adyen_api(@service, action, request, headers, @version)
     end
   end
+
+  class CheckoutLink < Service
+    def initialize(client, version = DEFAULT_VERSION)
+      @service = "Checkout"
+      @client = client
+      @version = version
+    end
+
+    def get(linkId, headers = {})
+      action = { method: 'get', url: "paymentLinks/" + linkId }
+      @client.call_adyen_api(@service, action, {}, headers, @version, true)
+    end
+
+    def update(linkId, request, headers = {})
+      action = { method: 'patch', url: "paymentLinks/" + linkId }
+      @client.call_adyen_api(@service, action, request, headers, @version, true)
```

Comment:
We return an error if you send `applicationInfo` for this request on TEST - just need to set the last param to `false`.